int32 count
wifi_sensor/wifi[] data